### PR TITLE
Client Version Update 30210604_1

### PR DESCRIPTION
**Pull request by zach2good**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

**Comment by TeoTwawki** [comment]
no textIDs or NPC IDs shifted?

**Comment by zach2good** [comment]
TODO:
- The zones clay asked me to hold off on
- There are a lot of duplicate IDs in the main city zones which trip up my tool, so I need to modify those by hand before I re-run it (only 5-10 zones, not so bad)
- I have no idea how to handle new items or npc shifts. Plz help.

**Review by claywar — APPROVED** [review]

File: scripts/zones/Alzadaal_Undersea_Ruins/IDs.lua [hunk 1]
```diff
@@ -10,31 +10,31 @@ zones[xi.zone.ALZADAAL_UNDERSEA_RUINS] =
 {
     text =
     {
-        NOTHING_HAPPENS         = 119, -- Nothing happens...
+        NOTHING_HAPPENS         = 119,  -- Nothing happens...
         ITEM_CANNOT_BE_OBTAINED = 6383, -- You cannot obtain the <item>. Come back after sorting your inventory.
         ITEM_OBTAINED           = 6389, -- Obtained: <item>.
         GIL_OBTAINED            = 6390, -- Obtained <number> gil.
         KEYITEM_OBTAINED        = 6392, -- Obtained key item: <keyitem>.
+        NOTHING_OUT_OF_ORDINARY = 6403, -- There is nothing out of the ordinary here.
```
Comment (kaincenteno) [inline scripts/zones/Alzadaal_Undersea_Ruins/IDs.lua:18]: thats a really weird shift, is it a duplicate of the same textID?

Comment (zach2good) [reply scripts/zones/Alzadaal_Undersea_Ruins/IDs.lua:18]: Good point: my new tool matches based on the first exact match it finds in the comments and actual text.
If `There is nothing out of the ordinary here.` appears multiple times, it'll pick the lowest id it sees for it. Functionally, this shouldn't make any difference :shrug:

Comment (kaincenteno) [reply scripts/zones/Alzadaal_Undersea_Ruins/IDs.lua:18]: awesome :)

**Comment by zach2good** [comment]
TODO: For the few zones that get chewed up by the update tool; I need to go into those zones and force those messages to see if (or by how much) they've shifted. Luckily, they're not zones with lots of texts, or with active important content (Abyssea etc.)

**Pull request merged**